src/main.zig

```diff
@@ -41,6 +41,21 @@ pub fn main() !void {
         if (gpa.detectLeaks()) std.posix.exit(1);
     };
 
+    // Exit the program on SIGINT signal. When running the browser in a Docker
+    // container, sending a CTRL-C (SIGINT) signal is catched but doesn't exit
+    // the program. Here we force exiting on SIGINT.
+    std.posix.sigaction(std.posix.SIG.INT, &std.posix.Sigaction{
+        .handler = .{
+            .handler = struct {
+                pub fn handler(_: c_int) callconv(.C) void {
+                    std.posix.exit(0);
```

Comment:
If it's worth it, we can create a global ?App and ?Server, and try to shutdown them cleanly.

The one argument that I can make for going through the trouble is that, on `deinit`, the app flushes the telemetry queue. Can also do it in a separate PR.